data/meta/vaults/1/0x20F6A2ec303dF7120eAe3346dE5aD877f80Beb64.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "vault",
+  "comment": "Balancer AuraBAL-f",
+  "hideAlways": false,
+  "depositsDisabled": false,
+  "withdrawalsDisabled": false,
+  "order": 341,
+  "migrationAvailable": false,
+  "allowZapIn": true,
+  "allowZapOut": true,
+  "retired": false,
+  "displayName": "Balancer AuraBAL",
```

Comment:
Seems like displayName / name details slightly differ from details show here, fine if intentional since the details added are far more readable: 

https://etherscan.io/address/0x20F6A2ec303dF7120eAe3346dE5aD877f80Beb64

Reply:
yah this is what its called on balancer's dapp